lib/node_modules/@stdlib/math/base/special/cbrt/lib/main.js

```diff
@@ -141,7 +141,7 @@ var WORDS = [ 0>>>0, 0>>>0 ]; // asm type annotations
 *
 * @example
 * var v = cbrt( -9.0 );
-* // returns ~-2.08
+* // returns -2.080083823051904
```

Comment:
This shouldn't need to be changed, as the `~` indicates an approximate value. Did you encounter a lint error here?

Reply:
Yup, a lint error appeared there

Reply:
The issue with providing an exact doctest value is that, if we change `cbrt` and it affects values within 1 ULP, we can get lint errors. Instead, for "inexact" values, we prefer documenting approximations.

Reply:
You mind pasting the lint error here in this thread?

Reply:
Now that I'm reverting this change, the lint error doesn't appear 😕